apps/emqx_rule_engine/src/emqx_rule_events.erl

```diff
@@ -1023,6 +1072,17 @@ columns_with_exam('client.check_authz_complete') ->
         {<<"timestamp">>, erlang:system_time(millisecond)},
         {<<"node">>, node()}
     ];
+columns_with_exam('client.check_authn_complete') ->
+    [
+        {<<"event">>, 'client.check_authz_complete'},
+        {<<"clientid">>, <<"c_emqx">>},
+        {<<"username">>, <<"u_emqx">>},
+        {<<"peerhost">>, <<"192.168.0.10">>},
```

Comment:
please add peername instead.
to unify, we should add peername to all events (which currently have peerhost), and consider peerhost deprecated.